test/Reserve.test.ts

```diff
@@ -254,19 +254,6 @@ describe('Reserve', () => {
         });
     });
 
-    describe('_getReserveAccumulatedAt()', () => {
```

Comment:
We should still test this function for edge cases. That's why coverage has decreased.